src/index.html

```diff
@@ -11,7 +11,123 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+  > 
+      <fieldset class="information"> 
+        <legend> Personal information </legend>
+        <label class="form-info">Surname:
+          <input type="text" name="lastname"
+         autocomplete="off" required
+          >
+        </label>
```

Comment:
```suggestion
        <label class="form-info">
          Surname:
          <input 
            type="text" 
            name="lastname"
            autocomplete="off"
            required
          >
        </label>
```

Reply:
If you started moving attributes to new line do it for every attribute ;)
Most often we do it with > 2 attr
also don't forget about indentation

fix it everywhere please

watch on checklist again more carefully :)
https://github.com/mate-academy/layout_html-form/blob/master/checklist.md